### DISCUSSION TITLE
Synch

### DISCUSSION THREAD

**Discussion by philips791** [kewisch/gdata-provider#350] [Support]
I am having issues with synchronizing using your add-on between my Google calendar and my Thunderbird calendar.  I am running v91.4.1 of Thunderbird (TB) & v91.0.2 of your add-on Provider for Google Calendar v91.0.2.  I thought that it was working fine a couple of days ago when I first implemented the add-on but that was primarily adding events from the Google calendar to the TB calendar.   But now none of the new events I enter into the TB calendar appear on my Google calendar.  Did a recent update of TB cause your add-on to fail?  Since this a bidirectional interface, events entered into the TB calendar should therefore be uploaded into the Google calendar?  I appreciate your assistance.  Thank you.

**Comment by kewisch** [comment]
Hey, not sure if this is still an issue for you? It should be bidirectional. If you go to the wiki there are some steps for debug logs, if you could enable them and share here I'd appreciate!
